frontend/src/components/events/EventList.js

```diff
@@ -1,11 +1,12 @@
 import React from "react";
 import EventSection from "./organisms/event_section";
 import "./index.css";
-import { findPastEndIndex } from "./utils";
+import { findFirstUpcomingIndex } from "./utils";
+import FeatureEvent from "./FeatureEvent";
 
 const EventList = ({ events }) => {
-  let pastEnd = findPastEndIndex(events)
-  if (pastEnd === events.length - 1) {
+  const firstUpcoming = findFirstUpcomingIndex(events);
+  if (firstUpcoming === undefined) {
```

Comment:
Its not good practice to check if something is equaled to `undefined`

I think a better way to structure this would be to put the `firstUpcoming === events.length - 1` first, then check if there is a `firstUpcoming` and then have this case go last

Reply:
https://github.com/the-difference-engine/ymim/blob/8954c9471285fef5aff5fadad2cdfa3b5632374a/frontend/src/components/events/EventList.js#L22